deps(destination-qdrant): 🐙 update dependencies [2026-03-24]

airbyteio opened the pull request:
# Update destination-qdrant

This PR was autogenerated by running `airbyte-ci connectors --name=destination-qdrant up_to_date --pull`

We've set the `auto-merge` label on it, so it will be automatically merged if the CI pipelines pass.
If you don't want to merge it automatically, please remove the `auto-merge` label.
Please reach out to the Airbyte Connector Tooling team if you have any questions or concerns.


## Operations

- Upgrade the base image to the latest version in metadata.yaml: Successful

- Update versions of libraries in poetry: Successful

- PATCH bump destination-qdrant version to 0.1.42: Successful

- Build destination-qdrant docker image for platform(s) linux/amd64, linux/arm64: Failed

- Create or update pull request on Airbyte repository: Successful

- Add changelog entry: Successful

> [!NOTE]
> **Auto-merge may have been disabled. Please check the PR status to confirm.**

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/airbytehq/airbyte?pullRequest=60734) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/airbytehq/airbyte?pullRequest=60734) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/airbytehq/airbyte?pullRequest=60734) it.</sub>

octavia-bot-hoard[bot] committed:
[up-to-date]Upgrade the base image to the latest version in metadata.yaml; Update versions of libraries in poetry; PATCH bump destination-qdrant version to 0.1.42; Create or update pull request on Airbyte repository; Add changelog entry

pnilan committed:
fix(destination-qdrant): widen Python constraint and update fastembed

Update Python requirement from ^3.9,<3.12 to ^3.10,<3.14 to support
the python-connector-base:4.1.0 image (Python 3.13). Update fastembed
from <0.3 to >=0.7,<0.9 to align with qdrant-client expectations and
gain Python 3.13 compatibility.

pnilan committed:
Revert "fix(destination-qdrant): widen Python constraint and update fastembed"

This reverts commit 2b2b77c63def2378ceba436db026a913ef3534e0.
